tests/bootstrap/init.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
I think we can probably just reference `require FRAMEWORK_PATH . '/tests/bootstrap/init.php';` directly from 
`tests/behat/serve-bootstrap.php` and `tests/bootstrap.php`.